etc/deckhand/config-generator.conf

```diff
@@ -0,0 +1,4 @@
+[DEFAULT]
+output_file = etc/deckhand/deckhand.conf.sample
```

Comment:
Glad we are generating an (oslo) configuration  - the other services should adopt this as its how we build help templates (tox -egenconfig) for the OpenStack services -- we could apply the same logic here.